udata/search/query.py

```diff
@@ -41,7 +41,12 @@ def execute_search(self):
             if self.sort:
                 url = url + f"&sort={self.sort}"
             for name, value in self._filters.items():
-                url = url + f"&{name}={value}"
+                param_value = value
+                # HACK: use the `.__class__.__name__` to avoid having to import `BoolFilter` here, as importing it at the top would make a import loop.
+                if self.adapter.filters[name].__class__.__name__ == "BoolFilter":
+                    # The search service uses 1 and 0 for booleans.
+                    param_value = 1 if value == "true" else 0
```

Comment:
Somehow on udata-search-service side, featured is not set to 0 or 1 [but 1 or 4](https://github.com/opendatateam/udata-search-service/blob/5bac31a90710c3281ffcf48198ed844aa3583041/udata_search_service/infrastructure/consumers.py#L36) (probably for poorly decided scoring reasons) :smiling_face_with_tear: